native/cocos/renderer/pipeline/custom/RenderCommonTypes.h

```diff
@@ -278,7 +278,7 @@ struct LightInfo {
     uint32_t level{0};
 };
 
-enum class DescriptorTypeOrder {
+enum class DescriptorTypeOrder : uint32_t {
```

Comment:
Why need changing to `uint32_t`? Does all enum type change to `uint32_t` now?

Reply:
This modification might not be necessary.
`load` specialization uses underlying_type, and default underlying type of enum class is int. Not all enum type need to change.